Vagrantfile

```diff
@@ -49,6 +49,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   config.vm.network "forwarded_port", guest: 9991, host: host_port, host_ip: host_ip_addr
+  config.vm.network "forwarded_port", guest: 9995, host: 9995, host_ip: host_ip_addr
```

Comment:
We should have thumbor be proxied by `run-dev.py`, similar to how tornado is, in which case we don't need this line.